app/models/article.rb

```diff
@@ -342,6 +342,11 @@ def tag_keywords_for_search
     tags.pluck(:keywords_for_search).join
   end
 
+  def long_publish_time
+    relevant_date = crossposted_at.present? ? crossposted_at : published_at
+    relevant_date.strftime("%e %B %Y at %I:%M%p %Z")
```

Comment:
this can be nil

Reply:
Thanks, resolved by adding safe navigation: `relevant_date&.strftime("%e %B %Y at %I:%M%p %Z")`